week_1/tests/test_answer.py

```diff
@@ -11,7 +11,7 @@
     week_1_pipeline,
 )
 
-DATA_FILE_PATH = "week_1/data/stock.csv"
+DATA_FILE_PATH = "../WEEK_1/data/stock.csv" 
```

Comment:
You might want to sync your fork with the original repo. Then made small changes on how these files are defined and how you'd run them (from the root folder). There could be other more significant changes for the comming projects as well

Reply:
updated the changes in challange PR and added vscode in gitignore
https://github.com/nemanasharma/corise-dagster/pull/3